src/faq/_faqAccordion.scss

```diff
@@ -0,0 +1,13 @@
+.FaqAccordion {
+  margin-top: $spacing-l;
```

Comment:
This also needs some margin on the left and right. Right now the answers scrape the sides of the screen if they're long enough.